lib/commands/network.js

```diff
@@ -168,7 +168,7 @@ commands.toggleLocationServices = async function toggleLocationServices () {
     if (api === 16) {
       // This version of Android has a "parent" button in its action bar
       seq.push(20); // down
```

Comment:
could we also put these magic numbers to constants, so then the comments, like `up, up` are not needed?